modular_zubbers/master_files/code/modules/holiday/holiday.dm

```diff
@@ -0,0 +1,86 @@
+#define PURPLE_LIGHT "#9F79F2"
+#define PURPLE_DARK "#6141A6"
+#define ORANGE_LIGHT "#F27F1B"
+#define ORANGE_DARK "#F24F13"
+#define GREY_LIGHT "#FFFFFF"
+#define GREY_DARK "#292929"
+
+/datum/holiday/halloween
+	name = HALLOWEEN
+	begin_day = 17
+
+/datum/holiday/halloween/New()
+	. = ..()
+	LAZYCLEARLIST(holiday_colors)
+	var/palette = rand(1, 15)
+	switch(palette)
+		if(1, 2)
+			holiday_colors += PURPLE_LIGHT
+			holiday_colors += PURPLE_DARK
+			holiday_colors += GREY_DARK
+			holiday_colors += ORANGE_DARK
+			holiday_colors += ORANGE_LIGHT
+			holiday_colors += GREY_LIGHT
```

Comment:
I personally feel lists like this are more readable, unless this was some sort of microoptimalization or is necessary due to the weird ways DM adds lists
```suggestion
			holiday_colors += list(
				PURPLE_LIGHT,
				PURPLE_DARK,
				GREY_DARK,
				ORANGE_DARK,
				ORANGE_LIGHT,
				GREY_LIGHT,
				)
```